requirements.txt

```diff
@@ -6,4 +6,5 @@ utm==0.4.0
 coverage>=4.5.1
 coveralls>=1.3.0
 dateparser==0.7.2
```

Comment:
Dateparser is now included in inicheck 0.7.9  if you want to remove

Reply:
Since it's a direct dependency of katana, I'd rather keep it as a requirement instead of relying on it being installed by inicheck.